### PR TITLE
Perception: use unordered_map to replace map for faster speed.

### PR DESCRIPTION
**Pull request by lianglia-apollo**

### CODE REVIEW

File: modules/perception/obstacle/camera/visualizer/frame_content.h [hunk 1]
```diff
@@ -174,33 +166,31 @@ class FrameContent {
...
   double current_fusion_timestamp_;
 
   // 4.ground truth
-  std::map<int64_t, GroundTruthContent> gt_caches_;
+  std::unordered_map<int64_t, GroundTruthContent> gt_caches_;
   double current_gt_timestamp_;
 
   // 5.image
-  std::map<int64_t, ImageContent> image_caches_;
+  std::unordered_map<int64_t, ImageContent> image_caches_;
```
Comment (eugenegx) [inline modules/perception/obstacle/camera/visualizer/frame_content.h]: caches is more convenient with map... I may use sorted property

Comment (eugenegx) [reply modules/perception/obstacle/camera/visualizer/frame_content.h]: Also, the camera visualization tool is under reconstruction.. thanks

Comment (lianglia-apollo) [reply modules/perception/obstacle/camera/visualizer/frame_content.h]: I changed them back to map as you will need the sorted property.

Comment (eugenegx) [reply modules/perception/obstacle/camera/visualizer/frame_content.h]: thanks!

**Pull request merged**